### DISCUSSION TITLE
How to change cursorline color?

### DISCUSSION THREAD

**Discussion by SaBenBurra** [LunarVim/LunarVim#3588] [Q&A]
How to change cursorline color?

**Comment by supermacro** [comment]
I tried `vim.cmd("highlight CursorLine guibg=#000000")` in the config.lua and this doesn't work :(

**Reply by TobaniEG** [reply]
https://www.lunarvim.org/docs/configuration/appearance/colorschemes#customizing-some-colors

or 
```lua
vim.api.nvim_create_autocmd("ColorScheme", {
	pattern = { "*" },
	callback = function()
		vim.api.nvim_set_hl(0, "CursorLine", { bg = "#ffffff"  })
	end
})
```
